Tasks/Zenchenko/task7-5.py

```diff
@@ -0,0 +1,25 @@
+cacheA = []
+cacheB = []
+cacheR = []
+
+
+def dec1(a, b):
+
+    def func(a1, b1):
+        return int(a1)*int(b1)
+    global cacheA
+    global cacheB
+    global cacheR
+    if cacheA == a and cacheB == b:
+        return cacheR
+    else:
+        cacheA = a
+        cacheB = b
+        cacheR = func(a, b)
+        return cacheR
+
```

Comment:
it's not the best approach, would be better to use a dict as a cache and some tuple of arguments as a key